Dockerfile.16.0

```diff
@@ -1,6 +1,6 @@
 FROM wonderfall/nginx-php:7.2
```

Comment:
7.2 is no longer available from wonderfall, change to:
`FROM wonderfall/nginx-php:7.3`

Reply:
I'm still able to pull just fine the 7.2 version.

Reply:
Yes, you are right, sorry about that!